nautobot/cloud/factory.py

```diff
@@ -86,7 +86,7 @@ def prefixes(self, create, extracted, **kwargs):
             if extracted:
                 self.prefixes.set(extracted)
             else:
-                self.prefixes.set(get_random_instances(Prefix))
+                self.prefixes.set(get_random_instances(model_or_queryset_or_lambda=Prefix, maximum=1))
```

Comment:
can this be reverted?

Reply:
I dont think so. since this is the sort_buffer_size error fix.

Reply:
Add todo comment